scripts/cdn_publish.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+version="$(git describe --abbrev=0 --tags)"
+mkdir -p cdn-dist/gravity/${version}
+
+cp dist/ui-lib/*.css cdn-dist/gravity/${version}
```

Comment:
You can just copy everything from this folder (i.e. `dist/ui-lib/*`). Things like the SVG (and it's associated JSON metadata file) should also be available to consumers.

In future Gravity releases we might also begin to add other files (e.g. fonts).

Reply:
Would we want to keep them separate to allow for us to define content-type?  Not sure how important that is for us